src/cmdlib.sh

```diff
@@ -272,15 +286,26 @@ EOF
     if [ -n "${ref_is_temp}" ]; then
         echo 'ref: "'"${ref}"'"' >> "${override_manifest}"
     fi
+
     if [ -d "${configdir}/overlay" ]; then
-        echo -n "Committing ${configdir}/overlay... "
-        ostree commit --repo="${tmprepo}" --tree=dir="${configdir}/overlay" -b "${name}-config-overlay" \
-          --owner-uid 0 --owner-gid 0 --no-xattrs --no-bindings --parent=none \
-          --timestamp "${git_timestamp}"
-        cat >> "${override_manifest}" <<EOF
-ostree-layers:
-  - ${name}-config-overlay
-EOF
+        local name="${name}-config-overlay"
+        commit_overlay "${name}" "${configdir}/overlay"
+        layers="${layers} ${name}-config-overlay"
+    fi
+    if [ -d "${ovld}" ]; then
```

Comment:
To be clear, if you have both a overlay and a overlay.d, both overlay and the overlays in overlay.d will get pulled in, yes?

Reply:
Yeah for now...I guess we might as well deprecate the other path.